lib/pry/helpers/completer_helpers.rb

```diff
@@ -0,0 +1,64 @@
+class Pry
+  module Helpers
+    module CompleterHelpers
```

Comment:
Not sure I'm a fan of this change, personally. Big-ish classes are not that bad.